File: frontend/src/types/report.ts
export interface Report {
  id: string;
  patientId: string;
  title: string;
  category: 'Radiology' | 'Lab' | 'Discharge' | 'Other';
  date: string;
  summary: string;
  status: 'Final' | 'Pending';
  fileUrl?: string;
}

File: backend/schemas/report.py
from pydantic import BaseModel
from typing import Optional

class ReportUploadResponse(BaseModel):
    file_path: str
    status: str

class ReportResponse(BaseModel):
    id: str
    patientId: str
    title: str
    category: str
    date: str
    summary: str
    status: str
    fileUrl: Optional[str] = None
